docs/tutorial/online-offline-events.md

```diff
@@ -78,3 +78,4 @@ _online-status.html_
 </body>
 </html>
 ```
+In Chrome and Safari, if the browser is not able to connect to a local area network (LAN) or a router, it is offline; all other conditions return true. So while you can assume that the browser is offline when it returns a false value, you cannot assume that a true value necessarily means that the browser can access the internet. You could be getting false positives, such as in cases where the computer is running a virtualization software that has virtual ethernet adapters that are always "connected." Therefore, if you really want to determine the online status of the browser, you should develop additional means for checking.
```

Comment:
Just copy pasting the paragraph (although mostly correct) doesn't quite solve the issue.

There is no point referring to "browsers" or "chrome" or "safari" as these concepts to not relate to Electron.  Also to make markdown more readable can you cut each line off at around 80 characters (most IDE's have a line there).

Should end up something like

```markdown
**NOTE:** If Electron is not able to connect to a local area network (LAN) or a router, it is considered offline; all other conditions return true. So while you can assume that the browser is offline when `navigator.onLine` returns a false value, you cannot assume that a true value necessarily means that the browser can access the internet. You could be getting false positives, such as in cases where the computer is running a virtualization software that has virtual ethernet adapters that are always "connected." Therefore, if you really want to determine the online status of the browser, you should develop additional means for checking.
```